tests/suites/0_stateless/02_function/02_0012_function_datetimes.sql

```diff
@@ -187,19 +187,19 @@ select '===EXTRACT===';
 
 select '===CMP===';
 
-select toDateTime64('2022-04-01 06:50:20.000')   = '2022-04-01 06:50:20.000';
-select toDateTime64('2022-04-01 06:50:20.000')   > '2022-04-01 04:50:20.000';
-select toDateTime64('2022-04-01 06:50:20.000')   < '2022-04-02 04:50:20.000';
+select toDateTime('2022-04-01 06:50:20')   = '2022-04-01 06:50:20';
+select toDateTime('2022-04-01 06:50:20')   > '2022-04-01 04:50:20';
+select toDateTime('2022-04-01 06:50:20')   < '2022-04-02 04:50:20';
 
 select '===INSERT===';
 drop table if exists ts;
-create table ts(a DateTime64, b DateTime, c Date, d Date32);
-insert into ts values(now(), now(), today(), today());
-select toDateTime64(a) = toDateTime64(b),  toDateTime32(a) = toDateTime(b),  toDate32(c) = toDate32(d), toDate(c) = toDate(d) from ts;
+create table ts(a DateTime_3, b DateTime, c Date);
+insert into ts values(now(), now(), today());
+select toDateTime(a) = toDateTime(b) from ts;
 drop table if exists ts;
 
 
-create table t(d32 datetime32, d64 datetime64);
+create table t(d32 datetime, d64 datetime_3);
```

Comment:
`datetime_3` data type is wired here, do we have plan to change it? @Veeupup @sundy-li

Reply:
Ok, I will change all the docs about date&time after it ready.

Reply:
@BohuTANG after #4982 merged, `date/datetime` is all ready.